f2/src/forms/__tests__/ContactInfoForm.test.js

```diff
@@ -36,7 +36,7 @@ describe('<ContactInfoForm />', () => {
       </MemoryRouter>,
     )
 
-    const inputNode = getAllByRole('textbox')[0]
+    const inputNode = getAllByLabelText('contactinfoPage.fullName')[0]
```

Comment:
the `getAllByRole` started complaining that it couldn't find a `textbox`.

Reply:
We don't have textboxes on contact info.... 🤔

Reply:
🤷‍♂